src/tlo/methods/rti.py

```diff
@@ -2314,6 +2314,15 @@ def report_daly_values(self):
         disability_series_for_alive_persons = df.loc[df.is_alive, "rt_disability"]
         return disability_series_for_alive_persons
 
+    def report_prevalence(self):
```

Comment:
Okay, I think I had 
      df = self.sim.population.props
         total_prev = len(
             df[(df['is_alive']) & (df['rt_inj_severity'] != 'none')]
         ) / len(df[df['is_alive']])

         return total_prev
     Originally, but I think Margherita implied that this would  include a consideration of recovery time. Do you think it's okay to use, still?

Reply:
Could possibly select cases using property rt_date_inj to make sure injury occurred in most recent time period!?

Reply:
Oh good idea! I've made that change now